### ISSUE TITLE
NeoPixel Led Strip not showing colours properly

### ISSUE THREAD

**Issue by angel41671** [HomeSpan/HomeSpan#877]

**Comment by HomeKidd** [comment]
> I have been swimming inside the pixel.cpp and pixel.h archives of the library and it seems what was described in the issue #781 has not been yet implemented... Am I right?
> 

Yep, PixelType is part of the [1.9.1 release](https://github.com/HomeSpan/HomeSpan/tree/release-1.9.1) and it (possibly) will be released soon. I'm also waiting for that official release :-)

**Comment by HomeSpan** [comment]
I anticipate releasing 1.9.1 to production in the next few days, pending final testing.

**Comment by angel41671** [comment]
All right! Thank you so much for the clarification, I didn't notice the "upcoming updates" page.

In the meantime I dared to stain your precious library to solve the little inconvenient. I share it just in case someone else is in a rush :/. By swapping the red and green values in the HSV setting colors in the pixel.h file I managed to get it working properly:

Original:
<img width="409" alt="Captura de pantalla 2024-07-03 a las 17 56 49" src="https://github.com/HomeSpan/HomeSpan/assets/54057871/80d88251-17e3-486b-81cd-c4fe1d16a414">

Modified:
<img width="408" alt="Captura de pantalla 2024-07-03 a las 17 57 06" src="https://github.com/HomeSpan/HomeSpan/assets/54057871/fa414eb1-1c21-4736-bfe7-a214640dea54">

It was the simplest way for me to solve it as I'm not a great coder.
Thanks once again for the marvelous software! :)
